cmd/wallet-web/src/pages/Signin.vue

```diff
@@ -18,6 +18,12 @@
   >
     <div class="flex flex-col flex-grow justify-center items-center">
       <!--Trustbloc Sign-up provider div -->
+      <toast-notification
+        v-if="systemError"
+        :title="i18n.errorToast.title"
+        :description="i18n.errorToast.description"
+        type="error"
+      ></toast-notification>
```

Comment:
What do you think about using camel case notation for user-defined element? For instance, `<ToastNotification />` instead of what we have right now. To me, it makes to easier to understand that this component is custom.

Reply:
https://v3.vuejs.org/guide/component-registration.html#component-names